### ISSUE TITLE
Tabs do not work

### ISSUE THREAD

**Issue by MagdaKrzyz** [BetaMasaheft/Dillmann#488]
In the DL the tabs that are used for encoding info., e.g. "language, transcription, transliteration, PoS" do not function. When I press on them nothing occurs.

**Comment by eu-genia** [comment]
they seem to be working to me

**Comment by MagdaKrzyz** [comment]
Try this entry for instance: https://betamasaheft.eu/Dillmann/lemma/L0c9820914f2d423aa3dd70699a2ae5e2.
add a TRaces meaninig and then insert language, transliteratiom, transcriptio etc.  It does not work for me. 
The tabs work when I add them in the Traces section on which I have already worked.

**Comment by eu-genia** [comment]
Let us look together when you are in the office, I do not understand what does not work. I just did what you suggested (did not save)
![image](https://github.com/user-attachments/assets/89dad28c-52df-49c5-9a90-5724230549fa)

**Comment by MagdaKrzyz** [comment]
Indeed, everything seems ok. But I tried on my laptop and again nothing. Good, let us do it when I come to the office.
